include/BNO055.hpp

```diff
@@ -160,7 +158,7 @@ class BNO055 {
     /**
      * Fetch data from the BNO055 using the custom i2c specification used by the device.
      *
-     * @param lowestAddress the lowest address to read the data from.
+     * @param[in] lowestAddress the lowest address to read the data from.
      * @param xBuffer a buffer to store the x data in.
```

Comment:
Yeah, these should be labeled `[out]` like you mentioned. Good catch
```suggestion
     * @param[out] xBuffer a buffer to store the x data in.
```